internal/appsec/appsec.go

```diff
@@ -30,14 +32,10 @@ const (
 // Default timeout of intake requests.
 const defaultIntakeTimeout = 10 * time.Second
 
-// Status returns the AppSec status string: "enabled" when both the appsec
-// build tag is enabled and the env var DD_APPSEC_ENABLED is set to true, or
-// "disabled" otherwise.
-func Status() string {
-	if enabled, _ := isEnabled(); enabled {
-		return "enabled"
-	}
-	return "disabled"
+// Enabled returns true when AppSec is up and running. Meaning that the appsec build tag is enabled, the env var
+// DD_APPSEC_ENABLED is set to true, and the tracer is started.
+func Enabled() bool {
+	return atomic.LoadPointer((*unsafe.Pointer)(unsafe.Pointer(&activeAppSec))) != nil
```

Comment:
Can we avoid using `unsafe`? If you want, use a plain `uint16` or something, and have `0` means off, and anything else mean on.

Reply:
Fixed by using a RWMutex instead in https://github.com/DataDog/dd-trace-go/pull/1042/commits/007399fa7b2f9deae49557e0366bba4634d3c67b